civihr_employee_portal/js/reports.js

```diff
@@ -306,18 +309,14 @@
         if (!this.filters) {
             return;
         }
-        CRM.$('#expose-filters-btn').bind('click', function(e) {
+        CRM.$('#js-filters-btn').on('click', function (e) {
             e.preventDefault();
-            CRM.$(this).addClass('hidden');
-            CRM.$('#collapse-filters-btn').removeClass('hidden');
-            CRM.$('#report-filters').removeClass('hidden');
-        });
-        CRM.$('#collapse-filters-btn').bind('click', function(e) {
-            e.preventDefault();
-            CRM.$(this).addClass('hidden');
-            CRM.$('#expose-filters-btn').removeClass('hidden');
-            CRM.$('#report-filters').addClass('hidden');
+
+            $('#js-filters-wrapper').toggleClass('panel--sliding-body');
```

Comment:
Is all this to create the collapse/uncollapse effect? If yes, you think that by using the `collapse` directive from the angular's ui-bootstrap module would help simplify things? For an example look [here](https://github.com/civicrm/civihr/blob/develop/uk.co.compucorp.civicrm.appraisals/views/dashboard.html#L74-L84)
```html
<span style="cursor: pointer;" ng-click="dashboard.filtersCollapsed = !dashboard.filtersCollapsed">
    <i ng-class="{ 'fa-angle-right': dashboard.filtersCollapsed, 'fa-angle-down': !dashboard.filtersCollapsed }"></i>
    Filter Appraisals
</span>
<div collapse="dashboard.filtersCollapsed">
```